HW6/eight_queens/HW6_Task_3.py

```diff
@@ -0,0 +1,21 @@
+"""
+Напишите функцию в шахматный модуль. Используйте генератор случайных чисел для
+случайной расстановки ферзей в задаче выше. Проверяйте различный
+ случайные  варианты и выведите 4 успешных расстановки."""
+
+from random import randint
+__all__ = ['get_pozis']
+
+
+def get_pozis(size=8):
+    list_posis = []
+    for i in range(size):
+        list_point = []
+        for j in range(2):
+            list_point.append(randint(1, size))
+        list_posis.append(list_point)
```

Comment:
Можно значительно проще
```suggestion
        list_posis.append([randint(1, size), randint(1, size)])
```